app/Website/routes/website-routes.php

```diff
@@ -11,4 +12,5 @@
 
     Route::get('/tags', [\App\Website\Http\Controllers\BlogTagsController::class, 'index']);
     Route::get('/blog-post-tags/{tag}', [BlogPostController::class, 'listByTag']);
+    Route::post('/contact', [ContactController::class, 'sendContactEmail'])->middleware('throttle:5,1');
```

Comment:
I believe the throttle is already set globally for all API routes

Reply:
I was able to send as many requests as I wanted, where is this defined?